File: Easy/remove_duplicates_from_sorted_array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums: return 0

        l = 0
        for i in range(len(nums)):
            if nums[i] != nums[l]:
                l+=1
                nums[l] = nums[i]

        return l+

File: Easy/Swift/remove_duplicates_from_sorted_array.swift
class Solution {
    func removeDuplicates(_ nums: inout [Int]) -> Int {
        guard !nums.isEmpty else { return 0 }
        
        var l = 0
        for i in 0..<nums.count {
            if nums[i] != nums[l] {
                l+=1
                nums[l] = nums[i]
            }
        }
        
        return l+1
    }
}
